acs-apiclient/AcsApiClientConfig.cs

```diff
@@ -87,14 +87,14 @@ public class AcsApiClientConfig
         internal readonly string ConsumerSecret;
 
         /// <summary>
-        /// Gets or sets the Access token
+        /// A token received upon authenticating
         /// </summary>
-        public string AccessToken { get; set; }
+        public string OAuthToken { get; set; }
 
         /// <summary>
-        /// Gets or sets the Access token secret
+        /// A secret received upon authenticating
```

Comment:
💡 "A token received upon completing the entire login process. Used in conjunction with the OAuthToken"